### DISCUSSION TITLE
How to get followers?

### DISCUSSION THREAD

**Discussion by goldentalent626** [community/community#145681] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi.
I am trying to get more followers.
What can i do right now?

**Comment by ritvak** [comment]
Engage with the GitHub community by following developers, contributing to open-source projects, starting discussions, and sharing your work through repositories.

**Reply by goldentalent626** [reply]
Thanks for your advice

**Reply by goldentalent626** [reply]
Hi, @ritvak.
I have one question.
What is your majority and what kind of programming are you learning now?
Thanks

**Reply by ritvak** [reply]
My major is Applied Computer Science, and I am currently focusing on software development. I am proficient in Java and currently learning advanced concepts of Java along with its frameworks like Spring Boot.

**Reply by goldentalent626** [reply]
Are you contributing to any project now?

**Reply by ritvak** [reply]
Not right now, but working on my personal projects to polish my skills

**Reply by goldentalent626** [reply]
Can you follow me and can we become friends?
Only if you dont mind.
